install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1445,6 +1445,18 @@ hubble:
             #     - destination_pod: ["frontend/nginx-975996d4c-7hhgt"]
             #     excludeFilters: []
             #     end: "2023-10-09T23:59:59-07:00"
+
+  # -- Emit v1.Events related to pods on detection of packet drops.
+  dropEventEmitter:
+    enabled: false
+    # --- Minimum time between emitting same events.
+    interval: 120s
```

Comment:
nit: how about `2m` instead?

Reply:
Personaly I always get confused for half a second about the 'm'. Is it minutes? Months? Milli-something? If you really want I can change it to `2m`.

Reply:
Changed to 2m.